### ISSUE TITLE
Mobile compatibility

### ISSUE THREAD

**Issue by wacko141za** [southern-wind/wind#29]
Changing of base layer and overlays on map is not possible

Opening the options menu is met with significant difficulty, only to reach a list of options that do not respond to touch

(Tested on wind-dev)

**Comment by jammin84** [comment]
@wacko141za what phone/tablet/etc did you use?

I could change layers on a HTC one. But it didn't display overlays as you've mentioned. i.e. nodes/links/etc.

It may be best that maps aren't shown for mobiles as I think the openlayers.org plugin may need upgrading from V2 to V3 for this to function usefully.

**Comment by wacko141za** [comment]
I tested using Chrome on a Samsung Galaxy S5

**Comment by wacko141za** [comment]
I've just noticed another issue on mobile. 
![screenshot_2016-01-28-10-49-47](https://cloud.githubusercontent.com/assets/2084416/12639424/f69268aa-c5ac-11e5-9c07-4d8fd33eff1c.png)

Wind home page and Network nodes pages scale properly, but node pages do not

**Comment by jammin84** [comment]
The real solution here is a total re-write with a responsive design.

Ideally the menu/ header / footer should disappear by becoming a small button on smaller screens.

But doubtful anyone has this kind of time/effort.

**Comment by wacko141za** [comment]
Sorry for being a hassle. Its just that often when I am out doing a node survey or an install, I will be constantly referring to wind on my phone. (It worked flawlessly on our previous version of wind)

**Comment by jammin84** [comment]
@wacko141za i agree it should be mobile friendly. I, and I'm sure many others do the same :wink:

**Comment by spinza** [comment]
@wacko141za  It wasn't flawless but yeah it worked.
